### ISSUE TITLE
References 

### ISSUE THREAD

**Issue by jraza19** [UBC-MDS/group29#29]
Please add your references here that you use so Rachel can compile them.

**Comment by sukh2929** [comment]
references help from Analise:  https://engineering.purdue.edu/~mark/puthesis/faq/cite-url/

**Comment by jraza19** [comment]
Note from Analise - use @manural or @misc

**Comment by rachelywong** [comment]
SCRIPT 4 - my part: 

- https://stackoverflow.com/questions/39187875/scikit-learn-script-giving-vastly-different-results-than-the-tutorial-and-gives for changing pos and neg labels of readmitted column
- store vals function from 571 Lab 4

**Comment by rachelywong** [comment]
**This is what I have so far, please take a look and comment any other references you have and where you used them if they are not in this list already:**

R - use citation():
- R

Python:
- Python3
- numpy
- pandas
- altair
- hashlib
- matplotlib
- IPython
- sklearn
- requests
- zipfile
- urllib
- io
- warnings

Others for Script 4:
- https://stackoverflow.com/questions/39187875/scikit-learn-script-giving-vastly-different-results-than-the-tutorial-and-gives for changing pos and neg labels of readmitted column
- store vals function from 571 Lab 4

**Comment by sukh2929** [comment]
reference from 573 Lab 1 and rest are already mentioned in above comment

**Comment by jraza19** [comment]
pandas profiling latest version needed 
altair_saver

**Comment by jraza19** [comment]
National Institue of Health(a). "Managing Diabetes", Accessed November 21, 2020. https://www.niddk.nih.gov/health-information/diabetes/overview/managing-diabetes

National Institue of Health(b). "Managing Diabetes", Accessed November 21, 2020. https://www.niddk.nih.gov/health-information/diabetes/overview/insulin-medicines-treatments

Vanderplas Jake. "Building rows of charts in for loop to create data dashboards #1281", 2019. Altair Github Repository. https://github.com/altair-viz/altair/issues/1281

**Comment by rachelywong** [comment]
> National Institue of Health(a). "Managing Diabetes", Accessed November 21, 2020. https://www.niddk.nih.gov/health-information/diabetes/overview/managing-diabetes
> 
> National Institue of Health(b). "Managing Diabetes", Accessed November 21, 2020. https://www.niddk.nih.gov/health-information/diabetes/overview/insulin-medicines-treatments
> 
> Vanderplas Jake. "Building rows of charts in for loop to create data dashboards #1281", 2019. Altair Github Repository. [altair-viz/altair#1281](https://github.com/altair-viz/altair/issues/1281)

@jraza19 where did you use these references? Just so I know where to put them in the final report!

**Comment by jraza19** [comment]
@rachelywong the NIH references were from the EDA under categorical variables.... We also reviewed of some categorical features against the target. We chose metformin and insulin as those are most commonly prescribed medications for diabetes management (NIHa) and Hemoglobin A1C levels is a common test for diabetes (NIHb).

the last one was for script 3 - EDA. so there is no reference to it in the final report. 

for pandas profiling we have to use the latest version 

@misc{pandasprofiling2019,
  author = {Simon Brugman},
  title = {{pandas-profiling: Exploratory Data Analysis for Python}},
  year = {2019},  
  howpublished = {\url{https://github.com/pandas-profiling/pandas-profiling}},
  note = "Version: 2.9, Accessed: November 2020" 
}
